tdrs-frontend/Dockerfile.staging

```diff
@@ -0,0 +1,42 @@
+# This Dockerfile is intended for use in deployments that target the
```

Comment:
I'm not sure of the order of operations here, but it would be nice if we could make it so we could use these files to set a local variable and then populate that variable based on the environment. That way, any other changes to the script in one place. Is that not possible in this circumstance?

Reply:
Due to the order Docker processes multi stage builds the only way I could see to do that would be to pass the variable in via docker build args. That is an alternative though, we could use a single Dockerfile and pass different environment variables to the docker build in CircleCI based on the current branch.